_posts/2021-05-13-paper-cardio.md

```diff
@@ -0,0 +1,23 @@
+---
+site: freiburg
+title: 'New Paper "Novel insights into the electrophysiology of murine cardiac macrophages: relevance of voltage-gated potassium channels"'
+doi: "10.1093/cvr/cvab126"
+tags:
+- paper
```

Comment:
the sfb1452 as the one that created this paper

Reply:
supporter then

Reply:
mentioned in the text